docs/en/02_Developer_Guides/18_Cookies_And_Sessions/01_Cookies.md

```diff
@@ -52,6 +56,20 @@ Cookie::force_expiry($name, $path = null, $domain = null);
 // Cookie::force_expiry('MyApplicationPreference')
 ```
 
+### Samesite attribute
+
+The `samesite` attribute is set on all cookies with a default value of `Lax`. You can change the default value by setting the `default_samesite` value on the
```

Comment:
```suggestion
The `samesite` attribute is set on all cookies (except the Session cookie) with a default value of `Lax`. You can change the default value by setting the `default_samesite` value on the
```

Reply:
The default for session cookies _is_ `Lax` though.

Reply:
Sorry, I phrased this wrong.  This paragraph should be something along the lines of changing `default_samesite` will change the default for all cookies except for the Session cookie, because it has separate config

Reply:
I can add it to that paragraph if you think it's needed, but I think that's covered by the `[info]` block below this
> Note that this _doesn't_ apply for the session cookie, which is handled separately. See [Sessions](/developer_guides/cookies_and_sessions/sessions#samesite-attribute).

Reply:
ok good enough